### PR TITLE
fix: sqlAlias on not partioned rollups

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: packages/cubejs-schema-compiler/src/adapter/PreAggregations.js [hunk 1]
```diff
@@ -107,8 +112,8 @@ export class PreAggregations {
 
   preAggregationDescriptionFor(cube, foundPreAggregation) {
     const { preAggregationName, preAggregation } = foundPreAggregation;
-    const name = foundPreAggregation.sqlAlias || preAggregationName;
```
Comment (ovr) [inline packages/cubejs-schema-compiler/src/adapter/PreAggregations.js:110]: `foundPreAggregation ` doesn't contain `sqlAlias`. `foundPreAggregation. preAggregation` should be used.

File: packages/cubejs-schema-compiler/src/adapter/PreAggregations.js [hunk 2]
```diff
@@ -593,18 +599,21 @@ export class PreAggregations {
     return this.query.cubeEvaluator.evaluatePreAggregationReferences(cube, aggregation);
   }
 
-  originalSqlPreAggregationTable(preAggregation) {
-    if (this.canPartitionsBeUsed(preAggregation)) {
-      return this.partitionUnion(preAggregation);
+  originalSqlPreAggregationTable(preAggregationDescription) {
```
Comment (hassankhan) [inline packages/cubejs-schema-compiler/src/adapter/PreAggregations.js:602]: Might be silly, but should the variable `preAggregationDescription` be called `preAggregationDescriptor` instead?

Comment (ovr) [reply packages/cubejs-schema-compiler/src/adapter/PreAggregations.js:602]: @hassankhan First, I renamed it to descriptor, but I realize that we have a method that called description.

Comment (hassankhan) [reply packages/cubejs-schema-compiler/src/adapter/PreAggregations.js:602]: Makes sense, thanks for clarifying 👍

**Pull request merged**